### PR TITLE
Books plugin documentation update

### PR DESCRIPTION
**Pull request by S-Haime**
Documentation for the books plugin.

Adding @joemull as a reviewer since you volunteered and @ajrbyers as this is your plugin.

Minor points:
- Monthly reporting is currently broken in the plugin. Issue is open, but this is therefor not included (but will be similar to regular metrics).
- What is the recommended size for books covers? Same as journal covers?
- Currently missing the Sphinx files and conf.py (I understand how it works in theory as its documented well enough + can see how it works on JW main and imports, but it might be good for me to see someone do it first, before messing about).

### CODE REVIEW

**Review by joemull — COMMENTED** [review]
Thanks @S-Haime--this is really good! I especially like the contextual detail you provide. Things like this show you are thinking through the questions people will have, which of course you are, but just wanted to say so explicitly because it is great:

> Make sure that the filename of the file uploaded is consistent and correct. Whilst Janeway will change the filename to the title internally, depending on the application used to open the document after download, the original filename might still be visible. Google Chrome is an example of an application that might still display the original filename in its reader toolbar, as displayed in the image below.

I added a few comments below.

In terms of building the documentation, I don't think we have a setup for RST on plugins yet. We'd have to configure that if we wanted it to show up on readthedocs.io. I can do that and help you with it @S-Haime so if that's what we want to do. Thoughts @ajrbyers?

Another option: Since we are phasing out RST, one easy short-term solution that keeps us from having to add more RST configuration is to just put this in a markdown file, something like `USER_GUIDE.md` at the top level of the repository. With an `images` folder replacing `nstatic` and a link from the main `README.md` file so people can read the docs on GitHub. Basic but functional. What do you think?

File: docs/Books.rst [hunk 1]
```diff
@@ -1 +1,284 @@
...
+- Affiliation
+    - This is a required field.
+- Email
+- Sequence
+    - This determines the order in which contributors are displayed and will be auto-filled (but can be edited). This is a required field.
+
+Formats
+~~~~~~~~~
+
+This is where manuscript files are uploaded. Various file types can be used, such as PDF, ePub and Mobi.
```
Comment (ajrbyers) [inline docs/Books.rst:145]: W3C refer to them as EPUB rather than ePub or ePUB so perhaps we should follow that convention?

**Review by ajrbyers — COMMENTED** [review]
This is great. As @joemull mentioned we are phasing rst out so it may be worth converting this to md. For this file, as there are no toctrees it should be relatively simple.

**Comment by joemull** [comment]
Thanks @ajrbyers.

@S-Haime I think that means we should do this:

> Another option: Put this in a markdown file, something like `USER_GUIDE.md` at the top level of the repository. With an `images` folder replacing `nstatic` and a link from the main `README.md` file so people can read the docs on GitHub.

Let me know if I can help. I will go ahead and give this PR the status of Changes Requested.

File: docs/Books.rst (hunk 1 above)
Comment (S-Haime) [reply docs/Books.rst:145]: Done :)

**Comment by S-Haime** [comment]
The .rst file can be retired at some point now :)

File: docs/nstatic/books_dashboardblock_highlight.png
[the remark this answers is not in the document]
Comment (S-Haime) [reply docs/nstatic/books_dashboardblock_highlight.png:1]: (I haven't addressed any of the images, as I imagine we'll want to look at this as part of the website building.)

File: docs/Books.rst [hunk 2]
```diff
@@ -1 +1,284 @@
...
+     - No
+   * - ISBN
+     - 
+     - No
+   * - Purchase URL
+     - 
+     - No
+    
+.. [#] Using a character encoding other than UTF-8 can cause bugs during imports or updates. `(What is character encoding?) <https://www.w3.org/International/questions/qa-what-is-encoding>`_. These apps save .csvs with UTF-8 by default: OpenRefine, LibreOffice, Google Sheets, and Apple Numbers. However! If you use Microsoft Excel, keep in mind some versions don’t automatically create .csv files with UTF-8 character encoding. This may cause punctuation and special characters to be garbled on import. So, when saving, look for the ‘.csv (UTF-8)’ option in the drop-down box.
+.. [#] Required due to a bug - we aim to fix this in the near future.
```
[the remark this answers is not in the document]
Comment (S-Haime) [reply docs/Books.rst:253]: Should be fixed now in .md

**Review by joemull — APPROVED** [review]

**Comment by joemull** [comment]
Thanks @S-Haime -- looks good to me. I added another commit to delete "Books.rst," if that's alright with you. Over to you for the second green tick @ajrbyers.

**Review by ajrbyers — APPROVED** [review]

**Pull request merged**